tests/data/sales.yaml

```diff
@@ -70,7 +70,7 @@ tables:
         "@id": "#items.item_id"
         datatype: int
         description: Unique item identifier
-        mysql:datatype: INT
+        mysql:datatype: INTEGER
```

Comment:
I'll remove this override from the test file - it is not needed or currently used in the test assertions.

Reply:
I made this change because `INT` triggered an error in felis logic after I fixed names `COLUMN_VARIANT_OVERRIDE` (which was totally broken and never tested).